### PR TITLE
feat: add auto-correct

### PR DESCRIPTION
**Pull request by suyanhanx**
This PR will contain the initial configuration of auto-correct and the first batch of corrections. 
We should consider the merge process based on it.

- fix: #217

### CODE REVIEW

**Comment by suyanhanx** [comment]
@gaocegege This is the preview PR after adding auto-correct. How do you think it looks?

**Review by gaocegege — APPROVED** [review]
cc @kemingy @VoVAllen

**Review by kemingy — APPROVED** [review]
/lgtm

You can fix all the lint errors.

**Comment by suyanhanx** [comment]
> /lgtm
> 
> You can fix all the lint errors.

Yes. That's what I'm going to do. This current failed check is to see how it works.

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

**Comment by suyanhanx** [comment]
It may be necessary to inform this change in the contribution guide.

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ This command generates static content into the `build` directory and can be serv
 Recommended to read the following part before you start to contribute the docs.
 
 - Chinese docs is under `/docs-zh`
-- VitePress markdown features [VitePress markdown](https://vitepress.vuejs.org/guide/markdown.html)
+- VitePress Markdown features [VitePress markdown](https://vitepress.vuejs.org/guide/markdown.html)
```
Comment (kemingy) [inline README.md:46]:
```suggestion
- VitePress Markdown features [VitePress Markdown](https://vitepress.vuejs.org/guide/markdown.html)
```

I guess the tool doesn't check the text inside the square brackets.

**Review by kemingy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (suyanhanx) [reply README.md:46]: Maybe. I'll check it.

**Comment by kemingy** [comment]
> It may be necessary to inform this change in the contribution guide.

SGTM

File: README.md (hunk 1 above)
Comment (suyanhanx) [reply README.md:46]: I have confirmed that the tool will check the text in square brackets.

Currently, the spell check of this tool is still experimental. Many problems need to be solved.

This line's situation is a bug. I'll open an issue for it.

**Comment by gaocegege** [comment]
Is this PR ready to merge?

**Comment by suyanhanx** [comment]
I think it is ready.

**Comment by gaocegege** [comment]
Thanks for your awesome contribution! :tada:

**Pull request merged**